### DISCUSSION TITLE
when open run.bat error

### DISCUSSION THREAD

**Discussion by thun-dar** [ehristoforu/DeFooocus#15] [Q&A] [answered]
![Screenshot 2024-05-30 223734](https://github.com/ehristoforu/DeFooocus/assets/171275465/0fa51de3-9a7c-4790-b7a1-81d9362f5056)

**Comment by Anistdev07** [comment, marked as answer]
Yes the same issue is occuring

**Reply by thun-dar** [reply]
I solved the problem install git in your pc it will download a folder named defooocus and all rum smoothly

**Reply by Anistdev07** [reply]
seems no luck still facing same issue

**Reply by Anistdev07** [reply]
finally it is running  smoothly thank you for that, You guys are doing fantastic job incase in future any bug or error occurs will  share the feed accoedingly.

**Reply by ChrisHamburg** [reply]
> I solved the problem install git in your pc it will download a folder named defooocus and all rum smoothly

Where did you find it and how did you do it?

**Reply by intoxicatedmind93** [reply]
> git

how did you solve this can you please help im facing the same issue.

**Comment by siddnaik60190** [comment]
Hello, I installed GIT, yet, it is showing same error while running Foocus. Please help.

[a comment by an automated account is not shown]
